Documentation/getting-started.md

```diff
@@ -39,3 +39,24 @@ Sample data is available for download from the [SlicerAutoscoperM Sample Data](/
 * Ankle BVR data - This data was provided by Michael Rainbow.
   * Three frames of movement are included in the sample data.
   * Three partial volumes are included in the sample data. The tibia, talus, and calcaneus are included.
+
+## Transformation Overview
+
+This section will contain a figure that shows the different coordinate systems used and the associated transformations between them.
+
+## Glossary
+
+This section defines terms that are commonly used by Autoscoper and SlicerAutoscoperM
+
+* **3D Computed Tomography (CT)**: A 3D volume obtained from x-rays.
+* **4D CT**: Like 3D CT but with added motion over time.
+* **Biplaner Videoradiography (BVR)**: Two calibrated x-ray sources that allow for the simultaneous collection of radiographs from two perspectives.
+* **Digitally Reconstructed Radiograph (DRR)**: Synthetic radiograph generated from 3D CT using real world camera parameters.
+* **Partial Volume (PVOL)**: An individual rigid body cropped from a CT scan.
```

Comment:
suggest moving PVOL definition up above DRR

Reply:
PVOL: Saved as a TIFF stack, the PVOL is a volumetric image data file.  A PVOL is a subset of 3DCT scan data as defined from a segmented rigid body, and cropped to its dimensions.